src/plugin/parseZone/index.js

```diff
@@ -0,0 +1,69 @@
+const REGEX_TIMEZONE_OFFSET_FORMAT = /^(.*)([+-])(\d{2}):(\d{2})|(Z)$/
```

Comment:
this would also allow something like `+44:00` - not a biggie, but maybe this could be changed to `^(.*)([+-])([0-2]\d{1}):([0-5]\d{1})|(Z)$` so it only allows possible timezones?

Reply:
Interesting one here!
So, `dayjs` does its own validation on this, but not sure at what point it would fail (I've seen it's mainly when you try to format or using a getter).

I think we need some direction from @iamkun here to advice on the best approach, mainly as if we update the matching pattern and it doesn't match, it would just fallback to the default constructor, whereas now it passes the offset to the utc plugin (which I would expect to deal with).